tools/wrap/wrap.py

```diff
@@ -67,7 +68,7 @@ def parse_command_line(self, args=None):
 
         # optional arguments
         arg_parser.add_argument("--language", "-lang", help="the language for the ELL module", choices=["python", "cpp"], default=self.language)
-        arg_parser.add_argument("--target", "-target", help="the target platform", choices=["pi3", "orangepi0", "pi3_64", "aarch64", "host"], default=self.target)
+        arg_parser.add_argument("--target", "-target", help="the target platform", choices=["pi0", pi3", "orangepi0", "pi3_64", "aarch64", "host"], default=self.target)
```

Comment:
I would put pi0 after pi3 and before orangepi0, for consistency

Reply:
Wow, I just found a more serious issue here than just order, I missed a quote on pi3! Thank you for pointing out the order issue here :-)